components/Model/OKN/CardContent/components/OKNInfo/OKNInfo.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useMemo } from 'react';
+import classNames from 'classnames';
+
+import { Info } from 'components/UI/Card/components/Info/Info';
+import { Label } from 'components/UI/Card/components/Label/Label';
+import { Icon } from 'components/UI/Icons';
+import { IconType } from 'components/UI/Icons/Icons.types';
+
+import styles from './OKNInfo.module.css';
+
+export type OKNInfoProps = {
+    number: string;
+    status: string;
+};
+
+export function OKNInfo({ number, status }: OKNInfoProps) {
+    const oknNumberInfo = useMemo(
+        () => [
+            {
+                name: 'Номер ОКН',
+                text: number,
+            },
+        ],
+        [number],
+    );
+
+    const isLost = status === 'Утраченный';
+
+    return (
+        <div className={styles.oknInfo}>
+            <div className={styles.oknInfo__wrapper}>
+                <div className={styles.oknInfo__text}>
+                    <div>
+                        {isLost && (
+                            <Label color="#e63223" backgroundColor="rgba(230, 50, 35, 0.24)">
+                                Утрачен
+                            </Label>
+                        )}
+                        <div
+                            className={classNames(styles.oknInfo__oknTitle, {
+                                [styles.oknInfo__oknTitle_lost]: isLost,
+                            })}
+                        >
+                            Объект культурного наследия федерального значения
+                        </div>
+                    </div>
+                    <Info infos={oknNumberInfo} nameColor="rgba(200, 165, 99, 0.56)" />
+                </div>
+                <div
+                    className={classNames(styles.oknInfo__oknLogo, {
+                        [styles.oknInfo__oknLogo_lost]: isLost,
+                    })}
+                >
+                    <Icon type={IconType.OKN} />
+                </div>
+            </div>
+            {/* <div className={styles.oknInfo__file}>
```

Comment:
Let's delete it?

Reply:
No, we still have document in design and in in strapi, I'm just not sure is it the same thing or not. But I wrote css and html for it, so it will be used in one of next PRs

Reply:
👌👌👌👌